### PR TITLE
Landing page.v2

### PR DESCRIPTION
**Pull request by xenia11**
## Rationale

Changed UI of the landing page

## Screenshots:

Screenshot_11

Screenshot_12
Screenshot_13

## Task Name

- Issue Title

- #96

## Linting Checklist
- [x ] No commented code
- [ x] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

**Review by engramar — APPROVED** [review]
Perfect @xenia11! Great start! Thanks!

File: web/src/pages/landing/Landing.jsx [hunk 1]
```diff
@@ -2,59 +2,43 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 import { Container, Grid, Paper, Button } from "@material-ui/core";
-import landingSvg from "res/img/women.svg";
+import landingSvg from "./../../res/img/WoCA logo.png";
```
Comment (deBubz) [inline web/src/pages/landing/Landing.jsx:5]: Can you use the new convention

keep it as `"res/img/WoCA logo.png";` should woork

File: web/src/pages/userDashboard/UserDashboard.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { LayoutUserDashboard } from "components/layout/Layout";
 import UserProfile from "./userprofile/UserProfile";
 import UserEvents from "./userevents/UserEvents";
 import UserCourses from "./usercourses/UserCourses";
+import Footer from "./../../components/footer/Footer";
```
Comment (deBubz) [inline web/src/pages/userDashboard/UserDashboard.jsx:8]: you dont need to traverse through the folders anymore
you can get to any folder in src as absolute path
`"./../../components/footer/Footer";`

**Review by deBubz — CHANGES_REQUESTED** [review]
Also try to make the pass the CI/CD as well

```
src/components/layout/Layout.jsx
  Line 3:8:  'Footer' is defined but never used  no-unused-vars

src/pages/landing/Landing.jsx
  Line 4:27:  'Paper' is defined but never used  no-unused-vars
```

**Comment by xenia11** [comment]
@deBubz Hi Bao, CICD works now. Should I push it to dev now?

**Review by deBubz — APPROVED** [review]

**Pull request merged**